CHANGES.md

```diff
@@ -1,3 +1,17 @@
+## Version 0.6.1
```

Comment:
@johnynek this isn't a release of the current master. It's just a backport to the `series/0.6.x` branch that should be equivalent to `0.6.0` but with the addition of #1173.